prisma/schema.prisma

```diff
@@ -101,6 +107,7 @@ model User {
 
   storyContributions StoryContribution[]
   authVerifications  AuthVerification[]
+  Quizs              Quiz[]
```

Comment:
Can you rename this to match the convention? Quizs to quizzes

Reply:
done